editor/editor_inspector.cpp

```diff
@@ -1857,32 +1857,56 @@ void EditorInspectorSection::_notification(int p_what) {
 						arrow_position.x = margin_start;
 					}
 					arrow_position.y = (header_height - arrow->get_height()) / 2;
-					if (has_children_to_show) {
+					if (can_click_unfold) {
 						draw_texture(arrow, arrow_position);
 					}
 					margin_start += arrow->get_width() + separation;
 				}
 
+				Ref<Font> font = get_theme_font(SNAME("bold"), EditorStringName(EditorFonts));
+				int font_size = get_theme_font_size(SNAME("bold_size"), EditorStringName(EditorFonts));
+				Color font_color;
+				if (object->has_method("_get_property_warning")) {
```

Comment:
```suggestion
				if (object->has_method("_get_property_warning") && !get_theme_color(String(object->call("_get_property_warning", related_enable_property)).is_empty()) {
```
Would allow to remove one condition.